### PR TITLE
fix #54431 decrypt.co

### LINKED ISSUE #54431: decrypt.co
### Issue URL (Annoyance)
 [https://decrypt.co/26693/bsns-chinachain-launches-globally](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdecrypt.co%2F26693%2Fbsns-chinachain-launches-globally)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.4 nightly 21
Browser: | Firefox
AdGuard driver: | TDI
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>AdGuard Base<br/><br/><b>Annoyances:</b><br/> Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental
Custom filter subscriptions: | https://raw.githubusercontent.com/bogachenko/fuckfuckadblock/master/fuckfuckadblock.txt

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#54431

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1357,6 +1357,8 @@ slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,itprotoday.com
 bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.com,swindonadvertiser.co.uk,wiltshiretimes.co.uk##.cta-component-full-width
 !
 belretail.by##.bannerAfterArticle
+decrypt.co##.sc-118zf0s-2
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/annoyances.txt:1360]: It looks like class names like this don't change (I've checked also on iPhone)

Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt:1360]: I don't see this class name.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1357,6 +1357,8 @@ slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,itprotoday.com
...
+decrypt.co##div[style^="position"][style*="relative"] > div[size="10"] + div.Veblq
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1361]: also cannot find

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 3]
```diff
@@ -49,6 +49,7 @@ elmundo.es,standaard.be,closermag.fr,elnortedecastilla.es,elpais.com,ibtimes.co.
 allianz.de,easybib.com,st.com,mauijim.com,citationmachine.net,idevice.ro,tagheuer.com,adevarul.ro,arris.com,beinsports.com,cnn.com,globalscape.com,hotnews.ro,lorealparis.rs,oxouk.com,salesforce.com###optanon
 !
 yatasbedding.com.tr##.n-cookies-container
+decrypt.co##.iDynzw
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:52]: this is exists

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1360]: <details>

</details>

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1361]: <details>

</details>

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt:1360]: ok

**Pull request merged**